int32 roll_actual
int32 pitch_actual

int32 roll_desired
int32 pitch_desired
